epan/dissectors/packet-ippusb.c

```diff
@@ -0,0 +1,100 @@
+/* packet-ippusb.c
+ * Routines for IPPUSB packet disassembly
+ *
+ * Jamie Hare <jamienh@umich.edu>
```

Comment:
><jamienh@umich.edu> [](start = 14, length = 19)

I would probably switch this to MSFT email?

Reply:
My microsoft email will be deleted at the end of my internship

---
In reply to: [442546781](https://github.com/tjamhar/wireshark/pull/1#discussion_r442546781) [](ancestors = 442546781)